.repositories-update-contributing.sh

```diff
@@ -14,7 +14,7 @@
 set -e
 set -u
 
-YQ_IMAGE="mikefarah/yq:3.3.2"
+YQ_IMAGE="docker.io/mikefarah/yq:3.3.2"
```

Comment:
no [Quay.io](https://quay.io/) alternative, as I recall we decided not to have docker images as part of the repo